worlds/dkc3/Regions.py

```diff
@@ -0,0 +1,882 @@
+        level_list[21],
+        level_list[22],
+        level_list[23],
+        level_list[24],
+        LocationName.bleaks_house_region,
+        LocationName.blizzard_region,
+        LocationName.glacial_grotto_region,
+    ]
+
+    for i in range(0, len(k3_levels)):
+        connect(world, player, names, LocationName.k3_region, k3_levels[i])
+
+    # Razor Ridge Connections
+    razor_ridge_levels = [
+        level_list[25],
+        level_list[26],
+        level_list[27],
+        level_list[28],
+        level_list[29],
+        LocationName.barboss_barrier_region,
+    ]
+
+    for i in range(0, len(razor_ridge_levels)):
+        connect(world, player, names, LocationName.razor_ridge_region, razor_ridge_levels[i])
+        
+    if False:#world.include_trade_sequence[player]:
+        connect(world, player, names, LocationName.razor_ridge_region, LocationName.cifftop_cache_region,
+                lambda state: (state.has(ItemName.wrench, player, 1)))
+    else:
+        connect(world, player, names, LocationName.razor_ridge_region, LocationName.cifftop_cache_region)
+
+    # KAOS Kore Connections
+    kaos_kore_levels = [
+        level_list[30],
+        level_list[31],
+        level_list[32],
+        level_list[33],
+        level_list[34],
+        LocationName.kastle_kaos_region,
+        LocationName.sewer_stockpile_region,
+    ]
+
+    for i in range(0, len(kaos_kore_levels)):
+        connect(world, player, names, LocationName.kaos_kore_region, kaos_kore_levels[i])
+
+    # Krematoa Connections
+    krematoa_levels = [
+        level_list[35],
+        level_list[36],
+        level_list[37],
+        level_list[38],
+        LocationName.rocket_rush_region,
+    ]
+
+    for i in range(0, len(krematoa_levels)):
+        connect(world, player, names, LocationName.krematoa_region, krematoa_levels[i],
+                lambda state: (state.has(ItemName.bonus_coin, player, world.krematoa_bonus_coin_cost[player].value * (i+1))))
+                
+    connect(world, player, names, LocationName.krematoa_region, LocationName.knautilus_region,
+            lambda state: (state.has(ItemName.krematoa_cog, player, 5)))
+
+
+def create_region(world: MultiWorld, player: int, active_locations, name: str, locations=None, exits=None):
+    # Shamelessly stolen from the ROR2 definition
+    ret = Region(name, None, name, player)
+    ret.world = world
+    if locations:
+        for locationName, locationData in locations.items():
+            loc_id = active_locations.get(locationName, 0)
+            if loc_id:
+                loc_byte   = locationData[0] if (len(locationData) > 0) else 0
+                loc_bit    = locationData[1] if (len(locationData) > 1) else 0
+                loc_invert = locationData[2] if (len(locationData) > 2) else False
+
+                location = DKC3Location(player, locationName, loc_id, ret, loc_byte, loc_bit, loc_invert)
+                ret.locations.append(location)
+    if exits:
```

Comment:
I'd suggest just removing this from your function since you aren't using this and are using a connect function instead. This is basically a much simpler version of the same thing